packages/fiori/src/DynamicPageTitle.ts

```diff
@@ -244,6 +269,10 @@ class DynamicPageTitle extends UI5Element {
 		return this.interactive ? "0" : undefined;
 	}
 
+	get hasSnappedTitleOnMobile() {
+		return isPhone() && this.snapped && this.snappedTitleOnMobile.length;
+	}
```

Comment:
if you agree with my comment above, this getter should be removed

Reply:
Resolved. :)